EIPS/eip-4844.md

```diff
@@ -391,8 +391,7 @@ The work that remains to be done to get to full sharding includes:
 
 This EIP also sets the stage for longer-term protocol cleanups:
 
-- It adds an SSZ transaction type which is slightly gas-advantaged (1000 discount) to nudge people toward using it,
```

Comment:
Let me know if I'm mistaken. I remember the gas-advantage consideration was removed.

Reply:
nice catch